LSP-css.sublime-settings

```diff
@@ -1,5 +1,10 @@
 {
 	"command": ["${node_bin}", "${server_path}", "--stdio"],
+	"schemes": [
+		"file",
+		"buffer",
+		"res"
+	],
```

Comment:
I know you just want to have things formatted for consistency but since the command above is not, it's now inconsistent. :)

And I prefer on one line anyway since it's a total waste of vertical space :)